wrapping/CMakeLists.txt

```diff
@@ -7,14 +7,14 @@ if(${ITK_WRAP_float} AND NOT ${ITK_WRAP_complex_float})
 endif()
 
 itk_wrap_module(Ultrasound)
-set(WRAPPER_LIBRARY_GROUPS
+set(WRAPPER_SUBMODULE_ORDER
+  itkCurvilinearArraySpecialCoordinatesImage
   itkAttenuationImageFilter
```

Comment:
I don't think we need to list all the files. We just need to list the ones which need special order (e.g. first). So removing everything from this list except `itkCurvilinearArraySpecialCoordinatesImage` might have the same effect.

Reply:
"Spectra1D" class wrappings are also expected to be order-dependent with `itkSpectra1DImageFilter.wrap` wrapping images of `std::list` pixels and subsequent wrappings consuming this wrapping. Per the warnings I mention in the summary, that `std::list` wrapping order does not seem to be working as intended 🙃 but the issue is not introduced in this PR.

I would prefer to leave the remaining wrapping order intact until that warning is resolved. Fine to remove list items after that.